test/e2e_node/eviction_test.go

```diff
@@ -600,6 +600,7 @@ func verifyEvictionOrdering(f *framework.Framework, testSpecs []podEvictSpec) er
 			}
 		}
 		gomega.Expect(priorityPod).NotTo(gomega.BeNil())
+
```

Comment:
unnecessary addition..

Reply:
ok.Sorry,i will pay attention to this condiction